apps/web/src/services/dashboards/widgets/_components/WidgetFrame.vue

```diff
@@ -49,7 +49,7 @@
                 </div>
                 <div class="footer-right">
                     <slot name="footer-right">
-                        <p-anchor v-if="(props.widgetLink || props.widgetLocation) && !props.printMode"
+                        <p-anchor v-if="(props.widgetLink || props.widgetLocation) && !props.printMode && !props.editMode"
```

Comment:
description: Hide 'full data' link in customize page

![스크린샷 2023-07-07 오전 8 42 08](https://github.com/cloudforet-io/console/assets/18563857/8f64d0fa-461e-43fe-a9e6-183da47c0985)